HLTrigger/Configuration/python/HLT_75e33/modules/hltL1TkMuons_cfi.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-hltL1TkMuons = cms.EDProducer("L1TkMuonProducer",
+L1TkMuons = cms.EDProducer("L1TkMuonProducer",
```

Comment:
@trtomei so now, [`hltL1TkMuons_cfi`](https://github.com/cms-sw/cmssw/pull/38442/files#diff-f33d006e90bc43f0a17a258db61e4c7de9e8562bc125a6ad03af52a860e34771) returns `L1TkMuons` (instead of `hltL1TkMuons`) ? That seems confusing. Can it be avoided?

Does HLT use the `L1TkMuons` from this cfi , or from the EDM input ?